app/src/main/java/com/cornellappdev/android/eatery/MainListFragment.java

```diff
@@ -86,12 +113,46 @@ public void initializeButtons(View rootView) {
                 R.id.brb};
         for (int id : viewIds) {
             Button tempButton = rootView.findViewById(id);
-            mButtons.put(id, tempButton);
+            mCampusButtons.put(id, tempButton);
             tempButton.setOnClickListener(this);
             changeButtonColor(R.color.blue, R.color.wash, tempButton);
         }
     }
 
+    public void initializeCollegeTownEateryButtons(View rootView) {
+        int[] viewIds = {
+                R.id.american,
```

Comment:
are these temporary? or predetermined by backend?

Reply:
These were just fillers I added the rest of them and just committed